composer.json

```diff
@@ -5,7 +5,7 @@
     "type": "project",
     "license": "GPL v2",
     "require": {
-        "php": ">=5.6",
+        "php": ">=5.6",    // IMPORTANT: If required PHP changed you have to update INSTALL.PHP and DOKU.PHP too.
```

Comment:
Comments are not allowed in composer.json

Reply:
@Klap-in 
> Comments are not allowed in composer.json

Thanks for your statement!

Do you have already read my collection of "reserch-informations" in this PR? And please take a look on the buildings protocols accessible via the given links.

**Update:** Oh sorry maybe my review was not stored ba my and you cannot read it.